benchmarks/load-testing/wrk/concurrency.sh

```
@@ -0,0 +1,35 @@
+echo "---------------------------------------------------------"
+echo " Running Primer $name"
+echo " wrk -H 'Host: $server_host' -H 'Accept: $accept' -H 'Connection: keep-alive' --latency -d 5 -c 8 --timeout 8 -t 8 $url"
+echo "---------------------------------------------------------"
+echo ""
+wrk -H "Host: $server_host" -H "Accept: $accept" -H "Connection: keep-alive" --latency -d 5 -c 8 --timeout 8 -t 8 $url
+sleep 5
+
+echo ""
+echo "---------------------------------------------------------"
+echo " Running Warmup $name"
+echo " wrk -H 'Host: $server_host' -H 'Accept: $accept' -H 'Connection: keep-alive' --latency -d $duration -c $max_concurrency --timeout 8 -t $max_threads \"$url\""
```

Shellcheck

[warning] duration is referenced but not assigned.

(SC2154)